src/app/core/auth.service.ts

```diff
@@ -116,4 +126,41 @@ export class AuthService {
   getGravatarImage(imgSize): string {
     return GRAVATAR.GET_GRAVATAR(this.email, imgSize);
   }
+
+  /**
+   * Identifies if the current logged in user has admin priviledges.
+   */
+  public hasAdminAccess(): boolean {
+    return this.group.getValue() > AUTH_GROUP.EDITOR;
+  }
+
+  /**
+   * Identifies if the current logged in user has editor priviledges.
+   */
+  public hasEditorAccess(): boolean {
+    return this.group.getValue() === AUTH_GROUP.EDITOR;
+  }
+
+  /**
+   * Identifies if the current logged in user has curator priviledges.
+   */
+  public hasCuratorAccess(): boolean {
+    return this.group.getValue() === AUTH_GROUP.CURATOR;
```

Comment:
Can we add a FIXME since this is a temporary measure fix?

Reply:
Perhaps something like this?

```suggestion
    // FIXME: This should be expanded to identify if the user has curator access to a specific collection
    return this.group.getValue() === AUTH_GROUP.CURATOR;
```